README.md

```diff
@@ -79,3 +79,63 @@ Press CTRL+C to quit
  * Debugger PIN: 113-666-066
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Testing
+
+### Directory structure
+
+Unit tests live in the same directory as the code that they test (e.g., `test_view_model.py` is in the same directory as `ViewModel.py`).
+
+Integration tests live in the `test_integration.py` file in the top level of the `todo_app` directory.
+
+### Terminal
+
+#### Run all unit and integration tests
+
+From the project root (`DevOps-Course-Starter`), run the following from your preferred shell:
+
+```shell
+$ pytest
```

Comment:
While you might be proactive about ensuring you've activated your virtual environment, I'm a fan of using `poetry run pytest` etc. so that we can be more confident that a new dev isn't accidentally running in their global environment